config/packages/fos_ck_editor.yaml

```diff
@@ -32,8 +32,7 @@ fos_ck_editor:
                 - ['Outdent', 'Indent', '-', 'Blockquote']
                 - ['Link', 'Unlink']
                 - '/'
-                - ['Source', '-', 'Image', 'Table', 'SpecialChar']
+                - ['Source', '-', 'Image', 'SpecialChar']
```

Comment:
Is OK that you removed Table functionality from CK editor?

Reply:
Fixed in 268abd1bae86f02635c43016b0523475fff3663c